docs/quality-assurance/test-policy

```diff
@@ -0,0 +1,62 @@
+# Purpose and Scope
+
+The purpose of this document is to communicate the policy for UKHO for testing of new software, updates, hardware and other related changes.
+
+## Why do we Test?
+
+* To build confidence in our systems
```

Comment:
Could be good to say that we test to help ensure that the user needs are met?

Reply:
Good point, I will add it.